file INSTALL cannot find "project_root/include/ktx.h" and "project_root/include/ktxvulkan.h"

My [open source project](https://github.com/FuXiii/Turbo) folder structure:
![image](https://user-images.githubusercontent.com/40132375/203457889-1cc8c66b-49d9-494b-a2d8-a77e02b7ac16.png)

After `build` the project and then `install` it:
```
...  
-- Up-to-date: E:/Turbo/install/bin/toktx.exe
-- Up-to-date: E:/Turbo/install/lib/ktx.lib
  CMake Error at thirdparty/KTX-Software/cmake_install.cmake:68 (file):
    file INSTALL cannot find "E:/Turbo/include/ktx.h": No error.
  Call Stack (most recent call first):
    thirdparty/cmake_install.cmake:38 (include)
    cmake_install.cmake:38 (include)
```
and then I check the `thirdparty/KTX-Software/cmake_install.cmake:68`:
```
if(CMAKE_INSTALL_COMPONENT STREQUAL "dev" OR NOT CMAKE_INSTALL_COMPONENT)
  file(INSTALL DESTINATION "${CMAKE_INSTALL_PREFIX}/include" TYPE FILE FILES
    "E:/Turbo/include/ktx.h"
    "E:/Turbo/include/ktxvulkan.h"
    )
endif()
```
it looks like I must create `include` folder in my project root and copy `ktx.h` and `ktxvulkan.h` from `project_root/thirdparty/KTX-Software/inlcude` into it.

Why?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -247,7 +247,7 @@ macro(commom_lib_settings lib write)
     endif()
 
     set_target_properties(${lib} PROPERTIES
-        PUBLIC_HEADER ${CMAKE_SOURCE_DIR}/include/ktx.h
+        PUBLIC_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/include/ktx.h
         VERSION ${PROJECT_VERSION}
         SOVERSION ${PROJECT_VERSION_MAJOR}
         XCODE_ATTRIBUTE_ENABLE_HARDENED_RUNTIME "YES"
@@ -447,7 +447,7 @@ macro(commom_lib_settings lib write)
         )
 
         get_target_property( KTX_PUBLIC_HEADER ${lib} PUBLIC_HEADER )
-        list(APPEND KTX_PUBLIC_HEADER ${CMAKE_SOURCE_DIR}/include/ktxvulkan.h)
+        list(APPEND KTX_PUBLIC_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/include/ktxvulkan.h)
         set_target_properties(${lib} PROPERTIES
             PUBLIC_HEADER "${KTX_PUBLIC_HEADER}"
         )

